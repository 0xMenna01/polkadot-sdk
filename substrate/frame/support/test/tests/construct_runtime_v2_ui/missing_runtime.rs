#[frame_support::construct_runtime_v2]
mod runtime {}

fn main() {}
