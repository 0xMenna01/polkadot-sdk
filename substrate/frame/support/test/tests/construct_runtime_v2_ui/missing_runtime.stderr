error: Missing `#[frame::runtime]`
 --> tests/construct_runtime_v2_ui/missing_runtime.rs:2:1
  |
2 | mod runtime {}
  | ^^^
